molgenis-metadata-manager/src/main/frontend/src/store/getters.js

```diff
@@ -64,6 +64,13 @@ export default {
     return rootAttributes.map(addChildren)
   },
   /**
+   *
+   */
+  getMappedByAttributes: (state, getters) => {
```

Comment:
You do not need the getters here.
can also write as `getMappedByAttributes: state => state.initialEditorEntityType.referringAttributes`